projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -71,6 +75,13 @@ export class GoTableComponent implements OnInit, OnChanges {
     this.renderTable();
   }
 
+  ngAfterViewInit(): void {
+    if (this.tableConfig.preselected) {
+      this.toggleSelectAll();
+      this.changeDetector.detectChanges();
```

Comment:
Added this because of https://medium.com/better-programming/expressionchangedafterithasbeencheckederror-in-angular-what-why-and-how-to-fix-it-c6bdc0b22787